csv_parser/csv_parser.py

```diff
@@ -56,10 +56,11 @@ def get_params_from_sheet(sheet):
         # Save all the column numbers that have 'Périmètre' in their name
         perimeter_columns = [i for i, x in enumerate(perimeter_row) if 'Périmètre' in str(x)]
         # Computing number of rows in table
-        # rows = df.iloc[7:, 0]
-        # Simply remove initial rows & total row
-        # ToDo: be more resilient to nan & \xa0 in full_df.iloc[8:,0] and compute nb with count?
-        rows = full_df.shape[0] - 8 - 1
+        # Find the row number of the first table header ('ID')
+        id_index = (full_df[0] == 'ID').idxmax()
+        id_column = full_df.loc[id_index+1:, 0]
+        # Count the number of rows in the ID column, without counting the last row (total)
+        rows = id_column.count() - 1
```

Comment:
Est-ce que ça serait pas mieux de faire 
```suggestion
        total_index = (full_df[0] == 'Total').idxmax()
        rows = total_index - id_index - 1
```

Reply:
typiquement, ça plante pas si quelqu'un écrit dans une cellule sous le tableau et c'est un poil plus simple je trouve